app/views/export/_answer.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="answer">
+  <%= answer.try(:value).try(:to_s).try(:html_safe) || "[No response]" %>
```

Comment:
I'm just wasting time pointing out that this version of ruby supports the safe navigation operator `&.` and noting that It's not worth the effort to update Aperta to use it.

Reply:
whoa, that's awesome. I didn't know!

Reply:
Well then I'm happy to have mentioned it!